ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -248,6 +248,9 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].profile is arista.avd.defined %}
    profile {{ ethernet_interfaces[ethernet_interface].profile }}
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].shape.rate is arista.avd.defined %}
+   shape rate {{ ethernet_interfaces[ethernet_interface].shape.rate }}{{ ' percent' if ethernet_interfaces[ethernet_interface].shape.rate | int < 100 }}
```

Comment:
I do not agree on the current approach. We should not implement logic like this in eos_cli_config_gen. It is also not correct.
It is much simpler and robust to just use a string in `shape.rate` like first proposed ex. `10 percent`, `1000 pps` or `99 kbps`. 
It is very platform depending which commands are supported, so this way we can support all of them with a simple data model.
From EOS:
```eos
(config-s-claus-if-Et1/1)#shape rate 8 ?
  kbps     Rate Unit Kbps (default)
  percent  Rate unit percent
  pps      Rate unit PPS
  <cr>     
```

Reply:
ethernet_interfaces:
  <Ethernet_interface_1 >:
    shape:
      rate: "< rate > kbps" | "1-100 percent" | "< rate > pps" , supported options are platform dependent
      burst_size: "< size > kbytes | mbytes | bytes"

Reply:
@tgodaA @carlbuchmann comments?

Reply:
Agreed. It was my mistake. I asked @c-po  to change. It's better if values and units are mentioned by the user.

Reply:
NP. I will change the PR to the above model.